tests/test_template.py

```diff
@@ -38,3 +40,18 @@ def test_find_shapes_from_one_slide(template):
 def test_find_shapes_index_out_of_range(template):
     with pytest.raises(IndexError):
         shapes = template._find_shapes(0, 'logo')
+
+
+def test_save_pdf(template):
+    try:
+        subprocess.run(['libreoffice', '--version'],
+                       stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)  # check if libreoffice is installed
+        output_path = 'test/test.pdf'
```

Comment:
Could this be handled using `tempfile.TemporaryDirectory()`, too? This would make the test even more readable, because you could skip the tear down (lines 53 to 55).

Reply:
In pytest, this is even simplert with the `tmp_path` fixture.
http://doc.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture